conans/test/util/tools_test.py

```diff
@@ -1814,6 +1815,80 @@ def work_on_project(tmp_folder):
         svn2.run('revert . -R')
         self.assertTrue(svn2.is_pristine())
 
+    def test_mixed_revisions(self):
+        project_url, _ = self.create_project(files={'myfile': "cc", 'another': 'aa'})
+        svn = SVN(folder=self.gimme_tmp())
+        svn.checkout(url=project_url)
+        with open(os.path.join(svn.folder, 'myfile'), "a") as f:
+            f.write('more')
+        svn.run('commit -m "up version"')
+        self.assertFalse(svn.is_pristine())
+
+
+class SVNToolTestsPristineWithExternals(SVNLocalRepoTestCase):
+
+    def _propset_cmd(self, relpath, rev, url):
+        return 'propset svn:externals "{} -r{} {}" .'.format(relpath, rev, url)
```

Comment:
I thought the definition is `svn:externals "-r148 http://svn.example.com/skinproj third-party/skins", so ' relpath' and 'revsion' changed

Reply:
I just recognized, that this is the new scheme (from Subversion 1.5). The old is still usable.

Reply:
These means, we have to check both schemes, because we have to find the revision & local path of it. We have to watch, because there are also some different whitepace possibilities:

```
$ svn propget svn:externals paint
http://svn.thirdparty.com/repos/My%20Project "My Project"
http://svn.thirdparty.com/repos/%22Quotes%20Too%22 \"Quotes\ Too\"
$
```